test/largescale/space/fn_base_slow_test.py

```diff
@@ -134,31 +150,45 @@ def test_setitem(fn):
         assert x[index] == -index
 
 
-@pytest.mark.xfail(reason='Expected to fail since inner scaling is not public')
```

Comment:
These xfails have been removed, now we test that the weighting is correct, but assume that the constant weighting is used, should be improved but works for all current test cases.